Support label value right align labels

jasonatball opened the pull request:
closes #442

donaldatball commented on lib/cosmos/tools/tlm_viewer/widgets/labelvalue_widget.rb:
```diff
@@ -20,20 +20,38 @@ class LabelvalueWidget < Qt::Widget
       setLayout(Qt::HBoxLayout.new)
       layout.setSpacing(0)
       layout.setContentsMargins(0,0,0,0)
+      layout.addStretch(1) if align.downcase == 'right'
```
Ok, so 'right' means label and value will both be squished to the right, 'left' means the label and value will both be squished to the left, but 'center' will push the label to the left and the value to the right.  Is that actually the intent?  I was thinking this would only adjust the alignment of the label part of the label/value pair.  That is to say, the value would always be right-aligned, and the label would aligned within the remaining available space based on this keyword.

Assuming that is the behavior you're actually looking for, though, the changes look ok to me.

jasonatball replied:
'center' is the current default behavior but I see what you mean that it's actually centering the space not the values.

jasonatball commented:
@ryanatball @donaldatball check out the new changes. I changed 'center' to actually push both the label and value together and center them. The default behavior I renamed to 'split'.

ryanatball commented:
:+1: Looks great
